### PR TITLE
[New] Augment reality to show tabletop scene

### PR DESCRIPTION
**Pull request by CalebRas**
## Description

This PR implements `Augment reality to show tabletop scene` in `Augmented Reality` category.
URL to README: [README](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-AugmentRealityToShowTabletopScene/Shared/Samples/Augment%20reality%20to%20show%20tabletop%20scene)

## Linked Issue(s)

- `swift/issues/4627`

## How To Test

- **Note**: As of creation, this PR requires `swift-toolkit-daily` to compile and run. 
- Tap on a flat, horizontal surface to place the scene.
- Move the camera around the scene to explore. 

## Screenshots

augment-reality-to-show-tabletop-scene

### CODE REVIEW

File: iOS/Info.plist [hunk 1]
```diff
@@ -33,6 +33,21 @@
...
+			<dict>
+				<key>NSIncludesSubdomains</key>
+				<true/>
+				<key>NSThirdPartyExceptionAllowsInsecureHTTPLoads</key>
+				<true/>
+			</dict>
+		</dict>
+	</dict>
+	<key>NSCameraUsageDescription</key>
+	<string>The camera used for augmented reality.</string>
```
Comment (des12437) [inline iOS/Info.plist:50]:
```suggestion
	<string>This app uses augmented reality to overlay imagery over your real-world environment. Camera access is required for this functionality.</string>
```
The camera usage description should be more specific.

File: Shared/Samples/Augment reality to show tabletop scene/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## How it works
+
+1. Create an `TableTopSceneView` with an anchor point, translation factor, clipping distance, and `SceneView`.
+2. For this sample, the `anchorPoint` is set to the center of the scene for best results. This will give the impression that the scene is centered on the location the user tapped.
+3. Set the `translationFactor` on the scene view such that the user can view the entire scene by moving the device around it. The translation factor defines how far the virtual camera moves when the physical camera moves.
+    * A good formula for determining translation factor to use in a tabletop map experience is **translationFactor = sceneWidth / tableTopWidth**. The scene width is the width/length of the scene content you wish to display in meters. The tabletop width is the length of the area on the physical surface that you want the scene content to fill. For simplicity, the sample assumes a scene width of 800 meters.
+4. Set the `clippingDistance` to clip the scene to the area you want to show.
+5. Create a `SceneView` with a scene. To allow you to look at the content from below, set the base surface navigation constraint to `none`.
```
Comment (des12437) [inline Shared/Samples/Augment reality to show tabletop scene/README.md:22]:
```suggestion
5. Create a `SceneView` with a scene. To allow you to look at the content from below, set the base surface navigation constraint to `unconstrained`.
```

**Review by des12437 — APPROVED** [review]

File: Shared/Samples/Augment reality to show tabletop scene/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Augment reality to show tabletop scene
+
+Use augmented reality (AR) to pin a scene to a table or desk for easy exploration.
+
+![Image of augment reality to show tabletop scene](augment-reality-to-show-tabletop-scene.png)
+
+## Use case
+
+Tabletop scenes allow you to use your device to interact with scenes as if they are 3D-printed model models sitting on your desk. You could use this to virtually explore a proposed development without needing to create a physical model.
```
Comment (pgruenler) [inline Shared/Samples/Augment reality to show tabletop scene/README.md:9]:
```suggestion
Tabletop scenes allow you to use your device to interact with scenes as if they are 3D-printed models sitting on your desk. You could use this to virtually explore a proposed development without needing to create a physical model.
```

File: Shared/Samples/Augment reality to show tabletop scene/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## How to use the sample
+
+You'll see a feed from the camera when you open the sample. Tap on any flat, horizontal surface (like a desk or table) to place the scene. With the scene placed, you can move the camera around the scene to explore. You can also pan and zoom with touch to adjust the position of the scene.
+
+## How it works
+
+1. Create an `TableTopSceneView` with an anchor point, translation factor, clipping distance, and `SceneView`.
```
Comment (pgruenler) [inline Shared/Samples/Augment reality to show tabletop scene/README.md:17]:
```suggestion
1. Create a `TableTopSceneView` with an anchor point, translation factor, clipping distance, and `SceneView`.
```

**Review by pgruenler — COMMENTED** [review]
Look good. Just a couple suggestions.

**Review by des12437 — APPROVED** [review]

**Pull request merged**